js/5etools-items.js

```diff
@@ -30,7 +30,7 @@ function d20plusItems () {
 		if (url && url.trim()) {
 			const handoutBuilder = playerMode ? d20plus.items.playerImportBuilder : d20plus.items.handoutBuilder;
 
-			if (url.trim() === `${DATA_URL}items.json`) {
+			if (url.trim() === `${DATA_URL}items.json` && !d20plus.importer.forceExternalRequests) {
```

Comment:
remove `&& !d20plus.importer.forceExternalRequests` -- `Renderer.item.pBuildList` uses `DataUtil.loadJSON` internally, so no special handling is required